data/transactions/logic/sourcemap.go

```diff
@@ -49,24 +51,35 @@ func GetSourceMap(sourceNames []string, offsetToLine map[int]int) SourceMap {
 		}
 	}
 
-	// Array where index is the PC and value is the line.
-	pcToLine := make([]string, maxPC+1)
+	// Backwards compatibility for `mapping` field.
+	oldPcToLine := make([]string, maxPC+1)
+	for pc := range oldPcToLine {
+		if line, ok := offsetToLine[pc]; ok {
+			oldPcToLine[pc] = MakeSourceMapLine(0, 0, line, 0)
+		} else {
+			oldPcToLine[pc] = ""
+		}
+	}
+	oldEncodedMapping := strings.Join(oldPcToLine, ";")
+
+	// Array where index is the PC and value is the line for `mappings` field.
+	prevSourceLine := 0
+	pcToLine := make([]string, maxPC)
```

Comment:
The above initializer has maxPC + 1, why the difference?

Reply:
Good catch, it should include the `maxPC` value